k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java

```diff
@@ -153,6 +151,8 @@ public class MessagingController implements Sync.MessageToSendCallback {
     public static final long INVALID_MESSAGE_ID = -1;
     public static final long SHARE_SIZE_THRESHOLD = 64000;
     public static final int SHARE_MAX_FILENAME_SIZE = 20;
+    public static final int CHECK_MESSAGE_ON_SERVER_INTERVAL = 5000;
```

Comment:
```suggestion
    public static final int CHECK_MESSAGE_ON_SERVER_INTERVAL_MILLIS = 5000;
```

I like having the unit in constant names. Or comment the unit somewhere.

Reply:
Constants were removed.